scaleway/resource_instance_security_group.go

```diff
@@ -412,15 +412,17 @@ func securityGroupRuleSchema() *schema.Resource {
 				Description: "Computed port range for this rule (e.g: 1-1024, 22-22)",
 			},
 			"ip": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				ValidateFunc: validation.SingleIP(),
-				Description:  "Ip address for this rule (e.g: 1.1.1.1). Only one of ip or ip_range should be provided",
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: func(i interface{}, s string) ([]string, []error) {
+					return validation.IsIPAddress(i, "server_ips")
```

Comment:
Why do you check for field `server_ips`?

Reply:
Fixed in https://github.com/scaleway/terraform-provider-scaleway/pull/577